README.md

```diff
@@ -6,6 +6,9 @@
 [![devDependency Status](https://david-dm.org/thelounge/lounge/dev-status.svg)](https://david-dm.org/thelounge/lounge?type=dev)
 
 # The Lounge
+<p align="center">
+	<img src="https://sr.ht/s8rn.png" width="728" />
```

Comment:
We can use a screenshot from our website instead of using third party, https://github.com/thelounge/thelounge.github.io/tree/master/img/screenshots